recipes/imp/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set version = "2.15.0" %}
+
+package:
+  name: imp
+  version: {{ version }}
+
+source:
+  fn: imp-{{ version }}.tar.gz
+  url: https://integrativemodeling.org/{{ version }}/download/imp-{{ version }}.tar.gz
+  sha256: be2c13c95e4347f2fa4fe4f845a109464e9b95c6a417e286ba57aa79a7cd1a16
+  patches:
+    - clang-6-cpp14.patch
+    - cmake-w32-deps.patch
+    - sfmap-swap.patch
+    - anon-export-fix.patch
+    - bc-clustering.patch  # [osx]
+    - imp-directories.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [win and py < 39]
+  detect_binary_files_with_prefix: True  # [not win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - pkg-config   # [not win]
+    - swig
+    - cmake
+    - ninja
+  host:
+    - python
+    - numpy
+    - rmf
+    - ihm
+    - boost-cpp
+    - hdf5
+    # We need both the libprotobuf library and the Python wrappers
+    - libprotobuf
+    - protobuf
+    # We only need the library; we don't use the OpenCV Python wrappers
+    - libopencv
+    - eigen
+    - fftw
+    - gsl
+    - libopenblas  # [linux]
```

Comment:
See our docs on how to specify blas libraries.

Reply:
We don't use BLAS, CBLAS, LAPACK or LAPACKE in this package. The only reason this is here is because gsl (which we do use) links against openblas on Linux, causing the overlinking check to fail. This is why I originally had a runtime dep on libopenblas, as per commit 5e088fb85. I don't see anything at https://conda-forge.org/docs/maintainer/knowledge_base.html#blas that addresses this - what would you suggest?

Reply:
If you are using `gslcblas`, then you should add `- libcblas`.